cli/pcluster/vpc_handler.py

```diff
@@ -0,0 +1,72 @@
+import boto3
+import functools
+
+from botocore.exceptions import ClientError, BotoCoreError
+
+
+def handle_client_exception(func):
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except (BotoCoreError, ClientError) as e:
+            raise RuntimeError(e.response["Error"]["Message"])
+
+    return wrapper
+
+
+class VpcHandler:
+    """
+    This class handles vpc automation related to pcluster.
+    """
+
+    @handle_client_exception
+    def __init__(self, aws_region_name):
+        """
+        Inizialize the VpcHandler with the specified region
+
+        :param aws_region_name: the region in which you want to use the VpcHandler
+        """
+        self.client = boto3.client("ec2", region_name=aws_region_name)
+        self.ec2 = boto3.resource("ec2", region_name=aws_region_name)
+
+    @handle_client_exception
+    def create(self):
+        """
+        Create a vpc for the given region name.
+
+        :return: the id of the created vpc
+        :raise RuntimeError: if some problems occurred during vpc creation
+        """
+        response = self.client.create_vpc(CidrBlock='10.0.0.0/16')
+        return response["Vpc"]["VpcId"]
+
+    @handle_client_exception
+    def set(self, vpc_id, name=None):
```

Comment:
"set" is a reserved word and it seems a method the sets something.
I'd use "setup" or "init" as method name.